src/SLCore/Service/Rules/Models/EffectiveRuleDetailsDto.cs

```diff
@@ -20,30 +20,24 @@
 
 using System.Collections.Generic;
 using Newtonsoft.Json;
+using SonarLint.VisualStudio.Core;
 using SonarLint.VisualStudio.SLCore.Common.Models;
 using SonarLint.VisualStudio.SLCore.Protocol;
+using Language = SonarLint.VisualStudio.SLCore.Common.Models.Language;
 
 namespace SonarLint.VisualStudio.SLCore.Service.Rules.Models;
 
 public record EffectiveRuleDetailsDto(
     string key,
     string name,
-    IssueSeverity severity,
-    RuleType type,
-    CleanCodeAttribute? cleanCodeAttribute,
-    CleanCodeAttributeCategory? cleanCodeAttributeCategory,
-    List<ImpactDto> defaultImpacts,
     Language language,
```

Comment:
Not relevant to this PR and those newly added languages are not supported in SLVS regardless

Reply:
The problem is that I have seen that one enum value has been added in the middle, which might change the int value of all upcoming values.
So I am not sure if through RPC the int value of the enum instead of the string one, which we might get wrong results.

Reply:
We don't care about int values. Enums are serialized as strings